content_script.js

```diff
@@ -112,31 +133,65 @@ function addButtonOnPlayer(container, regularNotShorts) {
 
   logger(`Adding ${type} screenshot button`);
   let btn = document.createElement("button");
-  let t = document.createTextNode("Screenshot");
+  
+  // Create SVG icon
+  btn.innerHTML = `
+    <svg width="100%" height="100%" viewBox="0 0 24 24" fill="currentColor">
+      <path d="M9 2L7 4H4C2.9 4 2 4.9 2 6v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2h-3l-2-2H9zm3 16c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5z"/>
+    </svg>
+  `;
+
+  // Set common button properties
+  btn.classList.add(btnClass);
+  btn.setAttribute("aria-label", "Take screenshot");
+  btn.setAttribute("title", "Take screenshot");
+  btn.style.cursor = "pointer";
 
   if (regularNotShorts) {
-    btn.classList.add("ytp-time-display");
+    // For regular YouTube player
     btn.classList.add("ytp-button");
+    
+    // Rely on YouTube’s native style and adjust the SVG
+    btn.style.display = "flex";
+    btn.style.alignItems = "center";
+    btn.style.justifyContent = "center";
+    btn.style.color = "white";
+    btn.style.padding = "0";
+    btn.style.margin = "0 4px";
+    
+    // Scale down the SVG slightly to match other icons
+    const svg = btn.querySelector("svg");
+    svg.style.width = "24px";
+    svg.style.height = "24px";
+    svg.style.pointerEvents = "none";    
+    
+    // Find the controls container and insert properly
+    const controls = document.querySelector(".ytp-right-controls");
+    if (controls) {
+      // Insert before the subtitle button
```

Comment:
Can you explain why you change the logic to insert the screenshot button as first control?